app/Repositories/Cts/ValidationPositionRepository.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace App\Repositories\Cts;
+
+use App\Models\Cts\Member;
+use App\Models\Cts\ValidationPosition;
+use Carbon\Carbon;
+use Illuminate\Support\Collection;
+
+class ValidationPositionRepository
+{
+    public function findByPositionId(int $id)
+    {
+        return ValidationPosition::find($id)->first();
+    }
+
+    public function findByPosition(string $position)
+    {
+        return ValidationPosition::where('position', 'like', "%{$position}%")->first();
+    }
+
+    public function getValidatedMembersFor(ValidationPosition $validationPosition)
+    {
+        return $this->formatValidatedMembers($validationPosition->members);
+    }
+
+    private function formatValidatedMembers($validatedMembers)
```

Comment:
Could do with typehinting the array for consistency with the rest of the file :)

Reply:
I'm passing a collection in here, which begs the question as to why this extraction was ever needed.

Removed in commit 83aa7d9b